BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/BottomNavigation/SkontoNavigationBarBottomAdapter.swift

```diff
@@ -48,4 +48,18 @@ public protocol SkontoNavigationBarBottomAdapter: InjectedViewAdapter {
      * - Parameter enabled: A bool value to reflect the state of the badge
      */
     func updateDiscountBadge(enabled: Bool)
+
+    /**
+     *  Set the saved amount text on the bottom navigation bar. Called when the saved amount needs to be displayed.
+     *
+     * - Parameter text: A string which contains the value of the saved amount.
+     */
+    func updateSavedAmount(with text: String?)
+
+    /**
+     *  Set the saved amount badge state. Called when the saved amount needs to be displayed.
+     *
+     * - Parameter enabled: A bool value to reflect the state of the saved amount badge.
+     */
+    func updateSavedAmount(enabled: Bool)
```

Comment:
[suggestion] Should we rename this into "displayInvoiceSkontoSavingsBadge(hidden: Bool)" because is not very clear for me what is doing for now.